.travis.yml

```diff
@@ -21,7 +21,7 @@ before_install:
   - cd coalahtml/_coalahtml
   - bower install
   - cd -
-  - sudo pip3 install -r requirements.txt
+  - sudo pip3 install -U -r requirements.txt
```

Comment:
Wouldn't just specific version in requirements do the trick and will much more transparent?